restcomm/restcomm.interpreter/src/test/java/org/restcomm/connect/interpreter/rcml/ParserTest.java

```diff
@@ -106,26 +113,50 @@ public void testParser() {
                 assertTrue(verb.attribute("playBeep").value().equals("false"));
                 parser.tell(next, observer);
                 verb = expectMsgClass(Tag.class);
+                
                 assertTrue(gather.equals(verb.name()));
                 assertTrue(verb.attribute("timeout").value().equals("30"));
                 assertTrue(verb.attribute("finishOnKey").value().equals("#"));
                 assertTrue(verb.hasChildren());
                 final List<Tag> children = verb.children();
                 Tag child = children.get(0);
+                
                 assertTrue(say.equals(child.name()));
                 assertTrue(child.attribute("voice").value().equals("man"));
                 assertTrue(child.attribute("language").value().equals("en"));
                 assertTrue(child.attribute("loop").value().equals("1"));
                 assertTrue(child.text().equals("Hello World!"));
                 child = children.get(1);
+                
                 assertTrue(play.equals(child.name()));
                 assertTrue(child.attribute("loop").value().equals("1"));
                 assertTrue(child.text().equals("https://127.0.0.1:8080/restcomm/audio/hello-world.wav"));
                 child = children.get(2);
+                
                 assertTrue(pause.equals(child.name()));
                 assertTrue(child.attribute("length").value().equals("1"));
                 parser.tell(next, observer);
+                verb = expectMsgClass(Tag.class);
+                
+                assertTrue(email.equals(verb.name()));
+                assertTrue(verb.attribute("from").value().equals("carlos@gmail.com"));
```

Comment:
better to use assertEquals to assert a comparison, it provides better error message in case teh test case is suddenly broken.